### DISCUSSION TITLE
Theme aka customColors

### DISCUSSION THREAD

**Discussion by redimongo** [Podcastindex-org/podcast-namespace#660] [Ideas]
This one is thanks to Adam on the podcastIndex 2.0 pod. I was speaking with Sam from Truefans and we thought of the following.

What if we could send in the RSS things like the following the idea is "a Podcast app/site might opt into allowing podcasts to change the sites logo, primary color and maybe even choose a template theme."

So we are playing around with the idea of something like either a JSON file, or an XML that can be added to the RSS to communicate to apps that want to allow theme customisation based on RSS.

Ideas:
- Transparent logo (clearly this needs to match a standard width and height I know if PodToo our standard height is 48px width can be up to 100px)
- Primary Color (we for example use MUI, so changing the primary color should be easy and not cause an issue with the site colors too much)
- Template Selection (could be standard names like : Clean, Modern, Sunset. For apps they just need to have their own matching names)

Basically we could do everything that Podpage does through RSS.

**How this could help Podpage (for example - please note I have not spoken to them).**
Imagine them reading the RSS file, getting the above and auto generating the website without the user having to select a theme or color or do any work at all.

**Podcast hosts**
We would require transparent logo and all the above settings when a customer on-boards or creates a new show. PodToo already offers a white label sites to our PodGrow and above customers, which already uses the above in to generate their white label site, we would just be opening into the RSS feed.

**Comment by daveajones** [comment]
There was a discussion of theme colors in the Issues section a couple of years ago (#194). I’m reading back through that now.

**Reply by redimongo** [reply]
yeah, one thing Sam and I have in common is we use MUI so technically we can let a user customise as much as they want as long as they follow MUI theme but that is JSON.

We most likely won't want to do that however, as the issue comes with the possibility of stuffing it up like white text and white background. But technically we can do checks on that JSON to make sure it's valid.

The other issue is if we base it off of MUI then we miss all those who use other style software like bootstrap etc.

So I think a more customised Podcast 2.0 `stylesheet` or `themeJSON` might work?
